Assets/Packages/com.studio23.ss2.achievementsystem/Runtime/Core/AchievementSystem.cs

```diff
@@ -0,0 +1,40 @@
+using Studio23.SS2.AchievementSystem.Data;
+using Studio23.SS2.AchievementSystem.Providers;
+using UnityEngine;
+
+namespace Studio23.SS2.AchievementSystem.Core
+{
+    public class AchievementSystem : MonoBehaviour
+    {
+        public static AchievementSystem instance;
```

Comment:
Public variable names must start with a capital letter. `public static AchievementSystem Instance` should be the correct one.